### PR TITLE
[DROOLS-5613] Disable Trusty pmml when jpmml is present

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @mariofusco @jiripetrlik

See 
https://issues.redhat.com/browse/DROOLS-5613

This PR
1) make uniform management of jpmml presence in legacy and trusty pmml
2) completely disable kie-pmml (legacy and trusty) if jpmml is present

### CODE REVIEW

**Review by danielezonca — APPROVED** [review]

**Comment by danielezonca** [comment]
jenkins retest this please

**Comment by danielezonca** [comment]
jenkins retest this please

File: kie-pmml-trusty/kie-pmml-evaluator/kie-pmml-evaluator-assembler/src/main/java/org/kie/pmml/evaluator/assembler/service/PMMLAssemblerService.java [hunk 1]
```diff
@@ -76,6 +79,16 @@ private static boolean isToEnable() {
         }
     }
 
+    private static boolean isjPMMLAvailableToClassLoader() {
+        try {
+            Thread.currentThread().getContextClassLoader().loadClass("org.kie.dmn.jpmml.DMNjPMMLInvocationEvaluator");
+            logger.info("jpmml libraries available on classpath, skipping kie-pmml parsing and compilation");
+            return true;
+        } catch (ClassNotFoundException e) {
+            return false;
```
Comment (jiripetrlik) [inline kie-pmml-trusty/kie-pmml-evaluator/kie-pmml-evaluator-assembler/src/main/java/org/kie/pmml/evaluator/assembler/service/PMMLAssemblerService.java:88]: Please add some log info that JPMML was not found. At least at debug level.

File: kie-pmml/src/main/java/org/kie/pmml/assembler/PMMLAssemblerService.java [hunk 2]
```diff
@@ -87,6 +90,16 @@ private static boolean isToEnable() {
         }
     }
 
+    private static boolean isjPMMLAvailableToClassLoader() {
+        try {
+            Thread.currentThread().getContextClassLoader().loadClass("org.kie.dmn.jpmml.DMNjPMMLInvocationEvaluator");
+            log.info("jpmml libraries available on classpath, skipping kie-pmml parsing and compilation");
+            return true;
+        } catch (ClassNotFoundException e) {
+            return false;
```
Comment (jiripetrlik) [inline kie-pmml/src/main/java/org/kie/pmml/assembler/PMMLAssemblerService.java:99]: Please add some log message.

**Review by jiripetrlik — APPROVED** [review]
Looks good to me. I have few minor comments.

**Pull request merged**